.github/config/rubocop_linter_action.yml

```diff
@@ -0,0 +1,12 @@
+check_name: "Rubocop Results"
```

Comment:
I think this file is unnecessary.

Reply:
I copied this from the v3-website, which had this file. Shall I remove and see if it still works?

Reply:
Please! And if so, remove it from there too :) I think this was only needed for a previous iteration of this, but I've since rewritten it.